### PR TITLE
Tincr checkpoint site name bug fix

### PR DESCRIPTION
**Pull request by ttown523**
In one part of the TINCR checkpoint export code, I was forgetting to change the name of an IOB site from "IOB_X0Y54" format to the package name "T10." This pull request fixes that bug.

### CODE REVIEW

**Comment by ttown523** [comment]
Can we approve this quick so we can get the changes out?

**Review by bradselw — APPROVED** [review]

**Comment by ttown523** [comment]
I can't merge this pull request...Can someone give me write access? I don't know how I lost it.

**Comment by bradselw** [comment]
You're added as an admin... but it says it's waiting for your response:
image

**Comment by ttown523** [comment]
Hmm, that is weird. Do you mind sending me the invite link again?

**Pull request merged**
